lesson2/README.md

```diff
@@ -102,6 +102,6 @@ Draw on the whiteboards a diagram of how a currently existing social media app m
 #### Part 2: Building on top of the app we showed you
 Your goal is to start making a quasi-to-do list app. We'll refer to it as the Simple To-do List app.
 
-1. This app will have two fragments and one activity. One fragments holds the to-do list, while the other fragment holds the settings page. The `activity` will have a button that lets you switch between `fragment`s.
-2. The to-do list `Fragment` should have five editable `textView`s. Add `onClick` listeners to them so that when you click on a to-do list item, an `alertDialog` will popup and ask for new text. The text the user inputs into the `alertDialog` will then replace the old text in the `textView`. Use a type of `layout` to organize the `textView`s. Don't worry if your app doesn't save the items you changed in your list when you switch fragments.
-3. The settings `fragment` will allow you to change the app's background color. Add three buttons that change the app's background color.
\ No newline at end of file
+1. This app will have two fragments and one activity. One fragments holds the to-do list, while the other fragment holds the settings page. The activity will have a button that lets you switch between fragments.
+2. The to-do list fragment should have five editable `textView`s. Add `onClick` listeners to them so that when you click on a to-do list item, an `alertDialog` will popup and ask for new text. The text the user inputs into the `alertDialog` will then replace the old text in the `textView`. Use a type of `layout` to organize the `textView`s. Don't worry if your app doesn't save the items you changed in your list when you switch fragments.
```

Comment:
I figured that if they do make data persist, then that's great, but for now to keep it simple and not worry about that